lsp-mode.el

```diff
@@ -7033,6 +7033,46 @@ STORE-PATH to make it executable."
       (set-file-modes store-path #o0700)
       store-path)
      ((f-exists? store-path) store-path))))
+
```

Comment:
Maybe start moving new util functions to a new lsp-utils.el file as suggested in #2167 ?

Reply:
I don't see this as a top priority issue and I don't want to spend time on it. I don't mind if someone wants to take on it.